gdc_ng_models/models/notifications.py

```diff
@@ -36,6 +40,12 @@ def to_dict(self):
             "dismissible": self.dismissible,
             "message": self.message,
             "level": self.level,
+            "start_date": self.start_date.isoformat()
+            if self.start_date is not None
+            else None,
+            "end_date": self.end_date.isoformat()
+            if self.end_date is not None
+            else None,
```

Comment:
There's a lot going on here. Could we separate this ternary statement into separate assignments and use the value in this return dictionary? Being separated into three lines makes it a little hard to read for me.